topics/userscript/index.md

```diff
@@ -0,0 +1,12 @@
+---
+aliases: userjs, greasemonkey, tampermonkey, violentmonkey
+created_by: Aaron Boodman
+display_name: Userscripts
+github_url: https://github.com/greasemonkey/greasemonkey
+released: March 28, 2005
+short_description: A kind of lightweight browser extension, used to modify webpages.
+topic: userscript
+wikipedia_url: https://en.wikipedia.org/wiki/Userscript
+related: WebExtension, JavaScript
+---
+A userscript is a kind of lightweight and less priveleged browser extension written in JavaScript used to modify webpages. Userscripts may use some API usually referred as [`GM_API`](https://wiki.greasespot.net/Greasemonkey_Manual:API) allowing them to bypass [same-origin policy](https://en.wikipedia.org/wiki/Same-origin_policy) and store the data in a separate storage. The technology was initially introduced in [GreaseMonkey](https://github.com/greasemonkey/greasemonkey) extension for [Firefox web browser](https://www.mozilla.org/firefox/), and then spread to other browsers, sometimes built-in, like in [Presto-based Opera](https://en.wikipedia.org/wiki/Presto_\(layout_engine\)), sometimes with help of extensions, like in [Chromium](https://www.chromium.org)-based browsers. The most famous catalogs of userscripts are [GreasyFork](https://greasyfork.org/) and [OpenUserJS](https://openuserjs.org/).
```

Comment:
`priveleged` => `privileged`

Could you add a comma after "API" and "GM_API", so it reads "Userscripts may use some API, usually referred as GM_API, allowing them..."?

I think you should drop the "a" in "store the data in a separate storage".

How about adding "the" to "in GreaseMonkey extension"?

Reply:
🤦. Done.